src/pages/signinLinkSent.js

```diff
@@ -23,7 +23,7 @@ class SignInLinkSent extends Component {
       res.statusCode = 302;
       res.setHeader('Location', '/signin');
       res.end();
-    } else {
+    } else if (router) {
```

Comment:
Why that change?

Reply:
I experience crashes because of router being null. It might have been just my limited nextjs knowledge :-) I will remove that as well.